docs/detectors/nuget.md

```diff
@@ -0,0 +1,15 @@
+# NuGet Detection
+
+## Requirements
+
+NuGet Detection depends on the following to successfully run: 
+
+- One or more `*.nuspec`, `*.nupkg`, `*.packages.config`, or `.*csproj` files.
+
+## Detection Strategy 
+
+NuGet Detection is performed by parsing any `*.nuspec`, `*.nupkg`, `*.packages.config`, or `*.project.assets` files found under the scan directory. By searching for all `*.nuspec,` `*.nupkg` files on disk the global NuGet cache gets searched which can include packages that are not included in the final build.
+
+## Known Limitations
```

Comment:
Can you also add a note about our underreporting dependencies that come from the .NET SDK. [See here][1]

[1]: https://github.com/microsoft/component-detection/blob/13f3e9f32c94bf6189fbd0bfbdf2e68cc60fccd9/src/Microsoft.ComponentDetection.Detectors/nuget/NuGetProjectModelProjectCentricComponentDetector.cs#L31-L185